packages/sw-background-sync-queue/demo/sw.js

```diff
@@ -16,20 +24,17 @@ let bgQueue = new goog.backgroundSyncQueue.BackgroundSyncQueue({callbacks:
 	},
 });
 
+const requestWrapper = new goog.runtimeCaching.RequestWrapper({
+  behaviors: [bgQueue],
+});
 
-self.addEventListener('fetch', function(e) {
-	if (e.request.url.startsWith('https://jsonplaceholder.typicode.com')) {
-		const clone = e.request.clone();
-		e.respondWith(fetch(e.request).catch((err)=>{
-			bgQueue.pushIntoQueue({
-				request: clone,
-			});
-			throw err;
-		}));
-	}
+const route = new goog.routing.ExpressRoute({
+  path: '/*',
+  origin: 'https://jsonplaceholder.typicode.com',
```

Comment:
@jeffposnick is it something that will change in near future?
cuz i right now it worked...
just curious